fplll/pruner/pruner.h

```diff
@@ -407,6 +675,26 @@ template <class FT> class Pruner
   */
   void load_coefficients(/*o*/ evec &b, /*i*/ const vector<double> &pr);
 
+  /**
+     @brief convert pruning coefficient from external to internal format.
+
+     Convert type, reverse the ordering, and select all coefficents
+
+     @param fb Output in internal format
+     @param pr Input in external format
+  */
+  void load_full_coefficients(/*o*/ vec &fb, /*i*/ const vector<double> &pr);
```

Comment:
Looks like these are conversion and not loading functions(?)

Reply:
Thanks for spotting. It's a ghost function not used ... removed it.